src/main/resources/liquibase/scripts/scriptsBot.sql

```diff
@@ -2,7 +2,9 @@
 
 -- changeset krassotin:1
 create table notification_task(
-                                  notification_task_id serial primary key ,
+
+                                  task_id serial primary key ,
```

Comment:
Поле в сущности у тебя называется id, поэтому тут нужно переименовать.
Также нужно указать тип int8 так как в сущности ты используешь Long:
```suggestion
                                  id int8 primary key ,
```